server/channels/app/support_packet.go

```diff
@@ -56,6 +59,25 @@ func (a *App) GenerateSupportPacket(c request.CTX) []model.FileData {
 		}
 	}
 
+	if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
+		pluginContext := pluginContext(c)
+		for _, id := range options.PluginPackets {
+			hooks, err := pluginsEnvironment.HooksForPlugin(id)
+			if err != nil {
+				c.Logger().Error("Failed to call hooks for plugin", mlog.Err(err), mlog.String("plugin", id))
+				warnings = append(warnings, err.Error())
+				continue
+			}
+			fileData, err := hooks.GenerateSupportData(pluginContext)
+			if err != nil {
+				c.Logger().Error("Failed to generate plugin file for support package", mlog.Err(err), mlog.String("plugin", id))
+				warnings = append(warnings, err.Error())
+			} else if fileData != nil {
```

Comment:
Should we stick to the same style as above?
```suggestion
				warnings = append(warnings, err.Error())
				continue
			}
			
			if fileData != nil {
```